pipeline_steps/deploy.groovy

```diff
@@ -65,6 +65,26 @@ def upgrade(Map args) {
   ) // conditionalStage
 }
 
+def upgrade_leap(Map args) {
```

Comment:
Well I forgot we had this script. Maybe it's worth re-using the script directly and patching it to start the leapfrog. Let's rollback this.